packages/api/src/webhooks/cannon.ts

```diff
@@ -474,26 +487,31 @@ export default class WebhookCannon {
     event: messages.WebhookEvent,
     resp: Response,
     startTime: [number, number],
-    responseBody: string
+    responseBody: string,
+    stream: DBStream,
+    params
   ) {
     try {
       const hrDuration = process.hrtime(startTime);
-      let encodedResponseBody = Buffer.from(responseBody).toString("base64");
+      const encodedResponseBody = Buffer.from(
+        responseBody.substring(0, 1024)
+      ).toString("base64");
 
       await this.db.webhookResponse.create({
         id: uuid(),
         webhookId: webhook.id,
         eventId: event.id,
+        event: event.event,
+        userId: webhook.userId,
```

Comment:
Btw do you plan on clearing the existing webhook_response objects or are we gonna be retrocompatible in some way? I'd be fine with just deleting them (or archiving if we wanna be safe), but I wonder if there might be problems if we keep the old objects in the DB and risk getting compatibility errors